banzai/munge.py

```diff
@@ -18,83 +21,12 @@ def do_stage(self, images):
         for image in images:
             telescope = dbs.get_telescope(image.telescope_id,
                                           db_address=self.pipeline_context.db_address)
-            # TODO: Currently we only support 1x1 Sinistro frames and only support 1x1 frames.
-            # TODO: 2x2 frames cannot use hard coded values because we read out more pixels.
+
             if 'sinistro' in telescope.camera_type.lower():
```

Comment:
The whole of the following block is pretty nasty. The nastiness comes from the intimate coupling of the data values and the structure of the logic. I would look at a possible rewrite splitting the data into one or two dicts, and then indexing into it as necessary within the logic (I can explain more in person if helpful).